docker/Dockerfile.Redis.6380

```diff
@@ -0,0 +1,8 @@
+FROM redis:7.0.8
+
+RUN mkdir -p /etc/deviceManager/redis
+COPY build/redis/redis.conf /etc/deviceManager/redis
+
```

Comment:
Don't we need to create the docker group and ad a user to it the same way as we do in other docker files?

Reply:
Default user changed.